calico_containers/pycalico/ipam.py

```diff
@@ -76,7 +76,7 @@ def _compare_and_swap_block(self, block):
             _log.debug("CAS Update block %s", block)
             try:
                 self.etcd_client.update(block.update_result())
-            except ValueError:
+            except EtcdValueError:
```

Comment:
I was worried that ValueError would catch more than the Compare Error that I expected, but looking again at the Exception hierarchy, I think you are right, we can afford to be more specific.